plugin/pkg/scheduler/schedulercache/cache_test.go

```diff
@@ -751,13 +806,25 @@ func benchmarkExpire(b *testing.B, podNum int) {
 	}
 }
 
-func makeBasePod(nodeName, objName, cpu, mem string, ports []v1.ContainerPort) *v1.Pod {
+type testingMode interface {
+	Fatalf(format string, args ...interface{})
+}
+
+func makeBasePod(t testingMode, nodeName, objName, cpu, mem, oir string, ports []v1.ContainerPort) *v1.Pod {
 	req := v1.ResourceList{}
 	if cpu != "" {
 		req = v1.ResourceList{
 			v1.ResourceCPU:    resource.MustParse(cpu),
 			v1.ResourceMemory: resource.MustParse(mem),
 		}
+		if oir != "" {
+			if len(strings.Split(oir, ":")) != 2 {
+				t.Fatalf("Invalid OIR string: %v", oir)
```

Comment:
@k82cn sorry, can you please elaborate your comment a bit? `t` is not necessary for what? do you mean that it could be `nil` as well or something else?